models/comments.models.js

```diff
@@ -0,0 +1,17 @@
+const db = require('../db/connection');
+
+exports.findCommentsByArticleId = (article_id) => {
+  return db.query(`SELECT 
+  * FROM comments 
+  WHERE article_id = $1
+  ORDER BY created_at ASC`, [article_id]).then(({rows}) => {
+    if (rows.length === 0) {
+      return Promise.reject({
+        status: 404,
+        msg: 'Article does not exist'
+      })
```

Comment:
Okay this might be fair but what if there are no comments but the article is valid? You have another test case on your hands here